test/controllers/declaration/OfficeOfExitControllerSpec.scala

```diff
@@ -75,20 +66,21 @@ class OfficeOfExitControllerSpec extends CustomExportsBaseSpec with OfficeOfExit
 
   "Office Of Exit Controller during standard declaration on GET" should {
 
-    "return 200 with a success" in new StandardSetUp {
-
+    "return 200 with a success" in {
```

Comment:
With removing traits you lost spec for supplementary and standard. The tests are the same, because working of choices we have tested in different place, but you left duplicated test.

Line 45 and 69

```
"return 200 with a success" in {

      val Some(result) = route(app, getRequest(uri))

      status(result) must be(OK)
    }
```

if you compare with test with this comment it's exactly the same..

So we have all tests duplicated, but with different name (standard and supplementary), so compilation and everything passed